QMLComponents/analysisform.cpp

```diff
@@ -943,6 +943,26 @@ void AnalysisForm::setShowRSyntax(bool showRSyntax)
 
 }
 
+void AnalysisForm::setShowRButton(bool showRButton)
+{
+	if (_showRButton == showRButton)
+		return;
+
+	_showRButton = showRButton;
+
+	emit showRButtonChanged();
+}
+
+void AnalysisForm::setDeveloperMode(bool developerMode)
+{
+	if (_developerMode == developerMode)
```

Comment:
what is this?

Reply:
developerMode is an AnalysisForm property, and must be set by the Analysis object when created

Reply:
yes I see that but why?

Reply:
So that the "Generate Wrappers" button appears only in Developer Mode.